testbenches/project/ad7606x/system_bd.tcl

```diff
@@ -56,9 +57,27 @@ create_bd_port -dir O sys_clk
 ad_connect sys_clk sys_cpu_clk
 
 set BA_AD7606X 0x44A00000
-set_property offset $BA_AD7606X [get_bd_addr_segs {mng_axi_vip/Master_AXI/SEG_data_axi_ad7606x}]
 adi_sim_add_define "AXI_AD7606X_BA=[format "%d" ${BA_AD7606X}]"
 
+set BA_SPI_REGMAP 0x44A00000
+adi_sim_add_define "SPI_AD7606_REGMAP_BA=[format "%d" ${BA_SPI_REGMAP}]"
```

Comment:
Move these definitions inside the if-else conditional part and add a definition verification part for these defined values in the test program before accessing these modules